processing/data_collection/gazette/spiders/sp_guaruja.py

```diff
@@ -8,30 +10,98 @@
 
 
 class SpGuaruja(BaseGazetteSpider):
+
     TERRITORY_ID = "3518701"
     name = "sp_guaruja"
     allowed_domains = ["guaruja.sp.gov.br"]
-    start_urls = ["http://www.guaruja.sp.gov.br/index.php/diario-oficial/"]
+    start_urls = ["http://www.guaruja.sp.gov.br/edicoes-diario-oficial/"]
+    start_date: dt.date
+    end_date: dt.date
+
+    def __init__(self, start_date=None, end_date=None):
+        self.start_date = dt.date.today().replace(day=1)
+        self.end_date = dt.date.today()
+
+        if start_date is not None:
```

Comment:
I guess these are done in base class

Reply:
It's already on #265. Maybe we can marge that one first, and then rebase this one?

Reply:
I mean here
https://github.com/okfn-brasil/querido-diario/blob/46050373fcf46e0808477640ceeaefd930f01557/processing/data_collection/gazette/spiders/base.py#L18

You should be able to call `super().__init__()`

Reply:
Hi! My bad I got confused because I did this for other spiders as well. Fixed.
The changes on the base class are on #265, as mentioned.